README.md

```diff
@@ -81,13 +81,13 @@ spack module lmod refresh
 ### Create container
 ```
 # See a list of preconfigured containers
-./create.py container -h
+spack stack create container -h
 
 # Create container spack definition (spack.yaml) in directory envs/<spec>.<config>
-./create.py container --config docker-ubuntu-gcc-openmpi --spec esmf
+spack stack create container docker-ubuntu-gcc-openmpi --app ufs-weather-model
```

Comment:
```suggestion
spack stack create container docker-ubuntu-gcc-openmpi.yaml --app ufs-weather-model
```

Reply:
I just checked and it now wants the extension as part of the container name. This was probably not intentional.

Reply:
Do you have an updated branch? This was a problem that I fixed. I can create a container with no `.yaml` suffix.

Reply:
I saw you just pushed that (https://github.com/NOAA-EMC/spack/pull/63/commits/c8e99bc1a533db38b4eac7da56f1c2fbb385f816)